.github/workflows/cockpit-lib-update.yml

```diff
@@ -0,0 +1,32 @@
+name: cockpit-lib-update
+on:
+  schedule:
+    - cron: '0 2 * * 4'
```

Comment:
Maybe please consider adding a comment what is the schedule for those less cron-literate?